### PR TITLE
SSJ - Scenario: Verifying Vacancy details

### PR DESCRIPTION
**Pull request by iamdez99**

### CODE REVIEW

**Review by iamdez99 — COMMENTED** [review]
Reviewed - juarezds

**Review by Mariachaudhry — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/SSJ_Runners/SSJ_Progression_RerunFailed.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package ServiceNow.PlatformBusinessApps.SSJ.SSJ_Runners;
+
+import io.cucumber.testng.AbstractTestNGCucumberTests;
+import io.cucumber.testng.CucumberOptions;
+
+@CucumberOptions(
+        features = {"@target/failed.txt"},
+        glue = {"ServiceNow.PlatformBusinessApps.SSJ.playwright.Steps", "ServiceNow.PlatformBusinessApps.SSJ.selenium.Steps", "Hooks"})
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/SSJ_Runners/SSJ_Progression_RerunFailed.java:8]: Making sure you do not need?
1. to generate the html report for failed test cases
2. add tags
3. dryrun

**Review by hamidnesar — COMMENTED** [review]
Hi Diego, 
I want to make sure you do not need:
1. to generate the html report for failed test cases
2. add tags
3. dryrun

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/SSJ_Runners/SSJ_Progression_RerunFailed.java (hunk 1 above)
Comment (iamdez99) [reply src/test/java/ServiceNow/PlatformBusinessApps/SSJ/SSJ_Runners/SSJ_Progression_RerunFailed.java:8]: You are correct about point #1 but 2 and 3 are not needed. This points only to the failed.txt and reruns only those scenarios.

**Review by alenaminsk — APPROVED** [review]

File: src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java
Comment (puzikovam2) [inline src/test/java/ServiceNow/PlatformBusinessApps/SSJ/playwright/Steps/ApplicantProfileSteps.java:1]: remove extra line 279

**Review by puzikovam2 — APPROVED** [review]
approved by @puzikovam2

**Review by georgeb79 — APPROVED** [review]

**Review by matakevin — APPROVED** [review]
LGTM

**Review by hamidnesar — APPROVED** [review]
Reviewed and approved!

**Pull request merged**